### PR TITLE
addcopy: add src validation

### PR DESCRIPTION
**Pull request by q384566678**
If the src is not specified, an error should be returned.

Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: cmd/buildah/addcopy.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 	}
 	name := args[0]
 	args = args.Tail()
+	if len(args) == 0 {
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/addcopy.go]: This should be "== 1".  The command should be `buildah add containerID SRC [[...] DEST]`  So both args[0] and args[1] should be provided.  Good catch though, that definitely was a hole that needed filling!

**Comment by q384566678** [comment]
ping @rhatdan

**Comment by rhatdan** [comment]
@q384566678 Can you update to the latest patches so travis will patch.

File: cmd/buildah/addcopy.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/buildah/addcopy.go]: Ooops!  Sorry about that @q384566678, I somehow missed the Tail() call a line above.  Ouch.  This is fine.

**Comment by TomSweeneyRedHat** [comment]
@q384566678 as @rhatdan you'll need to rebase your patch so you can pull in the the new .travis.yml from #572

**Comment by q384566678** [comment]
@rhatdan @TomSweeneyRedHat  rebased. PTAL.

**Pull request closed without merging**